app/moves/middleware/redirect-base-url.js

```diff
@@ -6,7 +6,7 @@ function redirectBaseUrl(req, res) {
 
   if (
     canAccess('moves:view:proposed') &&
-    currentLocation.location_type === 'prison'
+    currentLocation?.location_type === 'prison'
```

Comment:
is there no e2e test for this? If there is it will fail until we get babel 7 going with testcafe :(

Reply:
This file isn't included by the e2e tests so should be fine to use optional chaining. It's only when we include files directly in the e2es, like the services that create resources for test scenarios that we get caught out.

This middleware is only used within the app so should be fine :)